docs/source/flow_setup.rst

```diff
@@ -1,4 +1,22 @@
-Setup Instructions
+.. contents:: Table of contents
+
+1 - Local Installation
+    A. Dependencies
+    B. Sumo
+    C. Flow
+    D. Testing the Installation
+    E. Rllab-multiagent (optional)
+    F. Ray/RLlib (optional)
+    G. Getting started (rllab-multiagent)
+    H. Getting started (Ray/RLlib)
+    I. Custom configuration
+
+2- Remote desktop using Docker (recommended for testing purpose)
+    a. Installation
+    b. Notebooks and tutorial
```

Comment:
`.. contents:` will actually automatically generate the table of contents, so no need for this anymore

Reply:
@AboudyKreidieh Deleted it, will know for the next time ! Thanks for your help